state/runtime/evm/instructions_test.go

```diff
@@ -67,6 +117,30 @@ func testLogicalOperation(t *testing.T, f instruction, tests twoOperandsLogical)
 	}
 }
 
+type threeOperandsArithmetic []struct {
+	a              *big.Int
+	b              *big.Int
+	c              *big.Int
+	expectedResult *big.Int
+}
```

Comment:
Can we abstract this, so that operands are represented as the following:
```go
type operandsArithmetic struct {
   operands []*big.Int
   expectedResult *big.Int
}
```

That way we can get rid off `oneOperandsArithmetic`, `twoOperandsArithmetic` and `threeOperandsArithmetic`